front/src/routes/devices/style.css

```
@@ -296,3 +320,366 @@
+
+.exportCancelButton:disabled,
+.exportSubmitButton:disabled {
+  opacity: 0.55;
+  cursor: default;
+}
+
+.exportSpinning {
+  animation: exportSpin 1s linear infinite;
+}
+
+@keyframes exportSpin {
```

Stylelint (17.14.0)

[error] Expected keyframe name "exportSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)